omod/src/main/java/org/openmrs/module/pihapps/page/controller/labs/LabWorkflowPageController.java

```diff
@@ -1,23 +0,0 @@
-package org.openmrs.module.pihapps.page.controller.labs;
-
-import org.openmrs.Order;
-import org.openmrs.module.pihapps.orders.LabOrderConfig;
-import org.openmrs.module.pihapps.PihAppsConfig;
-import org.openmrs.module.pihapps.orders.OrderStatus;
-import org.openmrs.ui.framework.UiUtils;
-import org.openmrs.ui.framework.annotation.SpringBean;
-import org.openmrs.ui.framework.page.PageModel;
-
-import java.util.ArrayList;
-import java.util.List;
-
-public class LabWorkflowPageController {
```

Comment:
Removed this controller entirely, as there is nothing needed in the model beyond standard uiframework components